balboa-client/src/main/java/com/blist/metrics/impl/queue/MetricFileQueue.java

```diff
@@ -72,18 +84,26 @@ private void open() throws IOException {
         this.directory.mkdirs();
 
         String logName = basename + ".data";
-        fileStream = new FileOutputStream(new File(directory, logName), true);
-        stream = new BufferedOutputStream(fileStream);
-        reopenTime = now + MetricQueue$.MODULE$.AGGREGATE_GRANULARITY();
+        file = new File(directory, logName);
```

Comment:
Whoops that is a mistake.  I should be reusing file in the following line.  We also need a reference to it on close